src/index.html

```diff
@@ -13,6 +13,34 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card">
+      <img src="..\src\images\imac.jpeg" alt="Screen photo"
+          class="card__picture">
+
+      <p class="card__item--model">
```

Comment:
"--" is using for modifiers 
```suggestion
      <p class="card__item-model">
```